### ISSUE TITLE
Unable to use with Laravel Passport (and likely other packages)

### ISSUE THREAD

**Issue by bstump** [Nexmo/nexmo-php#171]
This package is used in Laravel's "laravel/nexmo-notification-channel package". Due to the Zend requirement in the composer file of the nexmo/nexmo-php package, an error is being thrown when I try to install this package:

"zendframework/zend-diactoros": "^1.3"

It looks like other packages were having this issue as well and were able to resolve it by allowing multiple versions of Zend Diactoros:

"zendframework/zend-diactoros": "^1.3|~2.0"

I have not pulled and tested myself, but perhaps you could take a look so I can use this with my other packages?

Please let me know. Thanks!

**Comment by gcshri** [comment]
Seeing this error:

````
 Problem 1
    - Conclusion: don't install laravel/nexmo-notification-channel v2.0.0
    - Conclusion: remove zendframework/zend-diactoros 2.1.1
    - Installation request for laravel/nexmo-notification-channel ^2.0 -> satisfiable by laravel/nexmo-notification-channel[2.0.x-dev, v2.0.0].
    - Conclusion: don't install zendframework/zend-diactoros 2.1.1
````

**Comment by arubacao** [comment]
@gcshri and @bstump : I had similar problems and fixed them in this branch: https://github.com/arubacao/nexmo-php/tree/feature/php7-update

This does the trick:

```
{
    "name": "arubacao/test",
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/arubacao/nexmo-php"
        }
    ],
    "require": {
      "nexmo/client": "dev-feature/php7-update"
    }
}
```

**Comment by josegonzalez** [comment]
We're also seeing this issue in cakephp/cakephp#13229.

**Comment by lornajane** [comment]
Thanks @josegonzalez, we'll keep you posted

**Comment by lornajane** [comment]
Should be fixed with the 1.8.1 release so I'll close the issue but please shout if you see anything else we should know about. Thanks all!
